executor/point_get.go

```diff
@@ -142,6 +155,9 @@ func (e *PointGetExecutor) Open(context.Context) error {
 
 // Close implements the Executor interface.
 func (e *PointGetExecutor) Close() error {
+	if e.runtimeStats != nil {
+		e.snapshot.DelOption(kv.CollectRuntimeStats)
+	}
```

Comment:
Does this snapshot will be recycled, otherwise it seems unnecessary to call `DelOption`?

Reply:
For the transaction with multiple DML statement, If we don't clean the stats after use and always use the same stats, the later DML statement use the stats will contain the runtime information of the before DML statement.